### PR TITLE
Upgrade to ``black==21.7b0``

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

**Comment by dwoz** [comment]
re-run all

**Comment by s0undt3ch** [comment]
re-run full all

**Comment by s0undt3ch** [comment]
Full test suite passed with expected(known) Arch and FreeBSD failures.
Resolving conflicts.

**Comment by s0undt3ch** [comment]
re-run full all

File: salt/cloud/libcloudfuncs.py [hunk 1]
```diff
@@ -336,7 +341,7 @@ def reboot(name, conn=None):
         __utils__["cloud.fire_event"](
             "event",
             "{} has been rebooted".format(name),
-            "salt-cloud" "salt/cloud/{}/rebooting".format(name),
+            "salt-cloudsalt/cloud/{}/rebooting".format(name),
```
Comment (Ch3LL) [inline salt/cloud/libcloudfuncs.py:344]:
```suggestion
            "salt/cloud/{}/rebooting".format(name),
```

File: salt/engines/junos_syslog.py [hunk 2]
```diff
@@ -275,19 +275,20 @@ def __init__(self, options):
             self.title = topics
             if len(topics) < 2 or topics[0] != "jnpr" or topics[1] != "syslog":
                 log.debug(
-                    'The topic specified in configuration should start with \
-                    "jnpr/syslog". Using the default topic.'
+                    "The topic specified in configuration should start with            "
```
Comment (Ch3LL) [inline salt/engines/junos_syslog.py:278]: I don't believe these spaces are needed.

File: salt/engines/junos_syslog.py [hunk 3]
```diff
@@ -275,19 +275,20 @@ def __init__(self, options):
...
             else:
                 for i in range(2, len(topics)):
                     if topics[i] not in data:
                         log.debug(
-                            "Please check the topic specified. \
-                              Only the following keywords can be specified \
-                               in the topic: hostip, priority, severity, \
-                                facility, timestamp, hostname, daemon, pid, \
-                                 message, event. Using the default topic."
+                            "Please check the topic specified.                         "
```
Comment (Ch3LL) [inline salt/engines/junos_syslog.py:286]: these spaces too.

File: salt/engines/junos_syslog.py [hunk 4]
```diff
@@ -337,8 +338,8 @@ def parseData(self, data, host, port, options):
                     raise Exception("Arguments in config not specified properly")
             else:
                 raise Exception(
-                    "Please check the arguments given to junos engine in the\
-                     configuration file"
+                    "Please check the arguments given to junos engine in the           "
```
Comment (Ch3LL) [inline salt/engines/junos_syslog.py:341]: and here spaces

File: salt/engines/napalm_syslog.py [hunk 5]
```diff
@@ -198,8 +198,8 @@ def __virtual__():
     if not HAS_NAPALM_LOGS or not zmq:
         return (
             False,
-            "napalm_syslog could not be loaded. \
-            Please install napalm-logs library amd ZeroMQ.",
+            "napalm_syslog could not be loaded.             Please install napalm-logs"
```
Comment (Ch3LL) [inline salt/engines/napalm_syslog.py:201]: Here too

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Comment by s0undt3ch** [comment]
Updated

File: salt/states/boto3_route53.py [hunk 6]
```diff
@@ -641,7 +641,7 @@ def rr_present(
 
     if Type is None:
         raise SaltInvocationError(
-            "'Type' is a required parameter when adding or updating" "resource records."
+            "'Type' is a required parameter when adding or updatingresource records."
```
Comment (Ch3LL) [inline salt/states/boto3_route53.py:644]:
```suggestion
            "'Type' is a required parameter when adding or updating resource records."
```

File: salt/states/netacl.py [hunk 7]
```diff
@@ -76,8 +76,8 @@ def __virtual__():
     else:
         return (
             False,
-            "The netacl state cannot be loaded: \
-                Please install capirca and napalm.",
+            "The netacl state cannot be loaded:                 Please install capirca"
```
Comment (Ch3LL) [inline salt/states/netacl.py:79]: extra space

File: salt/states/neutron_network.py [hunk 8]
```diff
@@ -37,8 +37,8 @@ def __virtual__():
         return __virtualname__
     return (
         False,
-        "The neutronng execution module failed to load:\
-                    shade python module is not available",
+        "The neutronng execution module failed to load:                    shade python"
```
Comment (Ch3LL) [inline salt/states/neutron_network.py:40]: extra space

File: salt/states/neutron_secgroup.py [hunk 9]
```diff
@@ -43,8 +43,8 @@ def __virtual__():
         return __virtualname__
     return (
         False,
-        "The neutronng execution module failed to load:\
-                    shade python module is not available",
+        "The neutronng execution module failed to load:                    shade python"
```
Comment (Ch3LL) [inline salt/states/neutron_secgroup.py:46]: extra space

File: salt/states/neutron_secgroup_rule.py [hunk 10]
```diff
@@ -37,8 +37,8 @@ def __virtual__():
         return __virtualname__
     return (
         False,
-        "The neutronng execution module failed to load:\
-                    shade python module is not available",
+        "The neutronng execution module failed to load:                    shade python"
```
Comment (Ch3LL) [inline salt/states/neutron_secgroup_rule.py:40]: extra space

File: salt/states/neutron_subnet.py [hunk 11]
```diff
@@ -55,8 +55,8 @@ def __virtual__():
         return __virtualname__
     return (
         False,
-        "The neutronng execution module failed to load:\
-                    shade python module is not available",
+        "The neutronng execution module failed to load:                    shade python"
```
Comment (Ch3LL) [inline salt/states/neutron_subnet.py:58]: extra space

File: salt/states/pkg.py [hunk 12]
```diff
@@ -2469,159 +2478,159 @@ def latest(
     **kwargs
 ):
     """
-    Ensure that the named package is installed and the latest available
-    package. If the package can be updated, this state function will update
-    the package. Generally it is better for the
-    :mod:`installed <salt.states.pkg.installed>` function to be
-    used, as :mod:`latest <salt.states.pkg.latest>` will update the package
-    whenever a new package is available.
+     Ensure that the named package is installed and the latest available
```
Comment (Ch3LL) [inline salt/states/pkg.py:2481]: I think this needs to be alligned with `"""`

File: salt/utils/dictdiffer.py [hunk 13]
```diff
@@ -101,42 +101,42 @@ def recursive_diff(past_dict, current_dict, ignore_missing_keys=True):
...
-
-.. code-block:: text
-
-            common_key1:
-              common_key2:
-                changed_key1 from '<old_str>' to '<new_str>'
-                changed_key2 from '[<old_elem1>, ..]' to '[<new_elem1>, ..]'
-            common_key3:
-              changed_key3 from <old_int> to <new_int>
+        Calculates a recursive diff between the current_dict and the past_dict
```
Comment (Ch3LL) [inline salt/utils/dictdiffer.py:104]: aligned with `"""`

File: tests/integration/modules/test_mac_brew_pkg.py [hunk 14]
```diff
@@ -74,29 +74,25 @@ def test_version(self):
...
                     ),
                 )
                 # make sure the version is accurate and is listed in the pkg_list
                 self.assertIn(
                     version,
                     str(pkg_list[ADD_PKG]),
                     msg=(
-                        "The {} version: {} is \
-                              not listed in the pkg_list: {}".format(
+                        "The {} version: {} is                               not listed"
```
Comment (Ch3LL) [inline tests/integration/modules/test_mac_brew_pkg.py:94]: extra space

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: salt/modules/mysql.py [hunk 15]
```diff
@@ -1044,7 +1044,7 @@ def free_slave(**connection_args):
 
     slave_cur.execute("stop slave")
     slave_cur.execute("reset master")
-    slave_cur.execute("change master to MASTER_HOST=" "")
+    slave_cur.execute("change master to MASTER_HOST=")
```
[the remark this answers is not in the document]
Comment (s0undt3ch) [reply salt/modules/mysql.py:1047]: Leaving this one as it was, don't want to possibly break stuff

**Comment by s0undt3ch** [comment]
re-run full all

**Comment by s0undt3ch** [comment]
re-run full all

File: salt/modules/boto_route53.py [hunk 16]
```diff
@@ -254,29 +254,29 @@ def zone_exists(
     """
     Check for the existence of a Route53 hosted zone.
 
-    .. versionadded:: 2015.8.0
+     .. versionadded:: 2015.8.0
```
Comment (Ch3LL) [inline salt/modules/boto_route53.py:257]: had this on the original review but should have specified for the entire doc for this method everything is shifted to the right. Shouldn't it align with the `"""` above?

File: salt/modules/boto_route53.py [hunk 17]
```diff
@@ -548,27 +548,27 @@ def get_record(
     """
     Get a record from a zone.
 
-    CLI Example:
+     CLI Example:
```
Comment (Ch3LL) [inline salt/modules/boto_route53.py:551]: same here, the entire doc for this function has been shifted. I believe it needs to align with the `"""` above.

File: salt/modules/boto_route53.py [hunk 18]
```diff
@@ -659,27 +659,27 @@ def add_record(
     """
     Add a record to a zone.
 
-    CLI Example:
+     CLI Example:
```
Comment (Ch3LL) [inline salt/modules/boto_route53.py:662]: same here this entire doc was shifted and needs to be moved back to align with `"""`

File: salt/modules/boto_route53.py [hunk 19]
```diff
@@ -772,21 +772,21 @@ def update_record(
...
-        argument instead.
+    retry_on_errors
+         Continue to query if the zone exists after an error is
+         raised. The previously used argument `retry_on_rate_limit`
+         was deprecated for this argument. Users can still use
+         `retry_on_rate_limit` to ensure backwards compatibility,
+         but please migrate to using the favored `retry_on_errors`
+         argument instead.
+
+     error_retries
```
Comment (Ch3LL) [inline salt/modules/boto_route53.py:783]: I don't think this part of the doc should have been shifted. It seems to shift some of these things correctly in these docs and some of them not.

File: salt/modules/boto_route53.py [hunk 20]
```diff
@@ -861,21 +861,21 @@ def delete_record(
...
```
Comment (Ch3LL) [inline salt/modules/boto_route53.py:872]: same here. shift back one

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: salt/modules/boto_route53.py [hunk 21]
```diff
@@ -262,21 +262,21 @@ def zone_exists(
...
```
Comment (Ch3LL) [inline salt/modules/boto_route53.py:273]: pretty sure this error_retries needs to be shifted back one, but the retry_on_errors is correct.

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Review by Ch3LL — DISMISSED** [review]

File: salt/cloud/libcloudfuncs.py [hunk 22]
```diff
@@ -336,7 +341,7 @@ def reboot(name, conn=None):
         __utils__["cloud.fire_event"](
             "event",
             "{} has been rebooted".format(name),
-            "salt-cloud" "salt/cloud/{}/rebooting".format(name),
+            "salt/cloud/{}/rebooting".format(name),
```
Comment (twangboy) [inline salt/cloud/libcloudfuncs.py:344]: Not sure what's going on here... but it doesn't look right

File: salt/cloud/clouds/opennebula.py [hunk 23]
```diff
@@ -893,8 +895,7 @@ def _require_disk_opts(*args):
         for arg in args:
             if arg not in disk_info:
                 raise SaltCloudSystemExit(
-                    "The disk {} requires a {}\
-                    argument".format(
+                    "The disk {} requires a {}                    argument".format(
```
Comment (garethgreenaway) [inline salt/cloud/clouds/opennebula.py:898]: Do we need all this white space here?

File: salt/modules/consul.py [hunk 24]
```diff
@@ -2282,7 +2281,7 @@ def acl_clone(consul_url=None, token=None, **kwargs):
         ret["ID"] = ret["data"]
     else:
         ret["res"] = False
-        ret["message"] = "Cloning ACL" "item {} failed.".format(kwargs["name"])
+        ret["message"] = "Cloning ACLitem {} failed.".format(kwargs["name"])
```
Comment (twangboy) [inline salt/modules/consul.py:2284]: Probably needs a space here

File: salt/modules/consul.py [hunk 25]
```diff
@@ -2382,7 +2381,7 @@ def event_fire(consul_url=None, token=None, name=None, **kwargs):
         ret["data"] = ret["data"]
     else:
         ret["res"] = False
-        ret["message"] = "Cloning ACL" "item {} failed.".format(kwargs["name"])
+        ret["message"] = "Cloning ACLitem {} failed.".format(kwargs["name"])
     return ret
```
Comment (twangboy) [inline salt/modules/consul.py:2385]: And here

File: salt/cloud/clouds/softlayer_hw.py [hunk 26]
```diff
@@ -418,8 +418,7 @@ def get_passwd():
 
 
 def list_nodes_full(
-    mask="mask[id, hostname, primaryIpAddress, \
-        primaryBackendIpAddress, processorPhysicalCoreAmount, memoryCount]",
+    mask="mask[id, hostname, primaryIpAddress,         primaryBackendIpAddress, processorPhysicalCoreAmount, memoryCount]",
```
Comment (garethgreenaway) [inline salt/cloud/clouds/softlayer_hw.py:421]: Seems like some extra white space here too.

File: salt/modules/inspectlib/collector.py [hunk 27]
```diff
@@ -257,7 +257,10 @@ def _save_payload(self, files, directories, links):
         for p_type, p_list in (
             ("f", files),
             ("d", directories),
-            ("l", links,),
+            (
+                "l",
+                links,
+            ),
```
Comment (twangboy) [inline salt/modules/inspectlib/collector.py:263]: This is dumb

File: salt/modules/linux_acl.py [hunk 28]
```diff
@@ -227,8 +228,8 @@ def modfacl(acl_type, acl_name="", perms="", *args, **kwargs):
 
     cmd = "setfacl"
     if recursive:
-        cmd += (
-            " -R"  # -R must come first as -m needs the acl_* arguments that come later
+        cmd += (  # -R must come first as -m needs the acl_* arguments that come later
+            " -R"
```
Comment (twangboy) [inline salt/modules/linux_acl.py:232]: Also a dumb change

File: salt/cloud/libcloudfuncs.py (hunk 22 above)
Comment (garethgreenaway) [inline salt/cloud/libcloudfuncs.py:344]: Is this change right?  It looks like it's changing the event message not just the formatting?

File: salt/config/schemas/esxvm.py [hunk 29]
```diff
@@ -139,8 +144,10 @@ class ESXVirtualMachineSerialSchemaItem(ComplexSchemaItem):
     )
     adapter = StringItem(
         title="Virtual Machine Serial Port Name",
-        description="Unique adapter name for virtual machine serial port"
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique adapter name for virtual machine serial port"
+            "for creation an arbitrary value should be specified"
```
Comment (garethgreenaway) [inline salt/config/schemas/esxvm.py:149]: Do we need a space at the start of the second line?

File: salt/config/schemas/esxvm.py [hunk 30]
```diff
@@ -161,8 +168,10 @@ class ESXVirtualMachineScsiSchemaItem(ComplexSchemaItem):
 
     adapter = StringItem(
         title="Virtual Machine SCSI Controller Name",
-        description="Unique SCSI controller name"
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique SCSI controller name"
+            "for creation an arbitrary value should be specified"
```
Comment (garethgreenaway) [inline salt/config/schemas/esxvm.py:173]: Need a space here on the second line too?

File: salt/config/schemas/esxvm.py [hunk 31]
```diff
@@ -194,8 +203,10 @@ class ESXVirtualMachineSataSchemaItem(ComplexSchemaItem):
     required = False
     adapter = StringItem(
         title="Virtual Machine SATA Controller Name",
-        description="Unique SATA controller name"
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique SATA controller name"
+            "for creation an arbitrary value should be specified"
```
Comment (garethgreenaway) [inline salt/config/schemas/esxvm.py:208]: Same here?

**Review by twangboy — CHANGES_REQUESTED** [review]
Reviewed to `salt/modules/lxd.py`... had to stop for a meeting

File: salt/config/schemas/esxvm.py [hunk 32]
```diff
@@ -219,16 +230,19 @@ class ESXVirtualMachineDiskSchemaItem(ComplexSchemaItem):
...
         default="GB",
         enum=("KB", "MB", "GB"),
     )
     adapter = StringItem(
         title="Virtual Machine Adapter Name",
-        description="Unique adapter name for virtual machine"
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique adapter name for virtual machine"
+            "for creation an arbitrary value should be specified"
```
Comment (garethgreenaway) [inline salt/config/schemas/esxvm.py:244]: Space needed here too?

File: salt/config/schemas/esxvm.py [hunk 33]
```diff
@@ -290,8 +304,10 @@ class ESXVirtualMachineInterfaceSchemaItem(ComplexSchemaItem):
     )
     adapter = StringItem(
         title="Virtual Machine Network Adapter",
-        description="Unique name of the network adapter, "
-        "for creation an arbitrary value should be specified",
+        description=(
+            "Unique name of the network adapter, "
+            "for creation an arbitrary value should be specified"
```
Comment (garethgreenaway) [inline salt/config/schemas/esxvm.py:309]: Same thing here.

File: salt/modules/inspectlib/collector.py (hunk 27 above)
Comment (twangboy) [reply salt/modules/inspectlib/collector.py:263]: Look at the lines above this to see what I mean

File: salt/cloud/libcloudfuncs.py (hunk 22 above)
Comment (s0undt3ch) [reply salt/cloud/libcloudfuncs.py:344]: yep, it's correct

Comment (s0undt3ch) [reply salt/cloud/libcloudfuncs.py:344]: yes, this is correct.

File: salt/config/schemas/esxvm.py (hunk 33 above)
Comment (s0undt3ch) [reply salt/config/schemas/esxvm.py:309]: This one looks good, no?

File: salt/config/schemas/esxvm.py (hunk 32 above)
Comment (s0undt3ch) [reply salt/config/schemas/esxvm.py:244]: Fixed

File: salt/modules/linux_acl.py (hunk 28 above)
Comment (s0undt3ch) [reply salt/modules/linux_acl.py]: Fixed

File: salt/modules/inspectlib/collector.py (hunk 27 above)
Comment (s0undt3ch) [reply salt/modules/inspectlib/collector.py]: Fixed

File: salt/modules/consul.py (hunk 25 above)
Comment (s0undt3ch) [reply salt/modules/consul.py:2385]: Fixed

File: salt/modules/consul.py (hunk 24 above)
Comment (s0undt3ch) [reply salt/modules/consul.py:2284]: Fixed

File: salt/client/ssh/ssh_py_shim.py [hunk 34]
```diff
@@ -242,8 +249,8 @@ def get_executable():
         stdout, _ = subprocess.Popen(
             [
                 py_cmd,
-                "-c"
-                "import sys; sys.stdout.write('%s:%s' % (sys.version_info[0], sys.version_info[1]))",
+                "-cimport sys; sys.stdout.write('%s:%s' % (sys.version_info[0],"
```
[the remark this answers is not in the document]
Comment (s0undt3ch) [reply salt/client/ssh/ssh_py_shim.py]: Fixed

File: salt/config/schemas/esxvm.py (hunk 31 above)
Comment (s0undt3ch) [reply salt/config/schemas/esxvm.py:208]: Fixed

File: salt/config/schemas/esxvm.py (hunk 30 above)
Comment (s0undt3ch) [reply salt/config/schemas/esxvm.py:173]: Fixed

File: salt/config/schemas/esxvm.py (hunk 29 above)
Comment (s0undt3ch) [reply salt/config/schemas/esxvm.py:149]: Fixed

File: salt/cloud/clouds/softlayer_hw.py (hunk 26 above)
Comment (s0undt3ch) [reply salt/cloud/clouds/softlayer_hw.py]: Fixed

File: salt/cloud/clouds/opennebula.py (hunk 23 above)
Comment (s0undt3ch) [reply salt/cloud/clouds/opennebula.py]: Fixed

File: salt/modules/junos.py [hunk 35]
```diff
@@ -1210,30 +1210,34 @@ def install_config(path=None, **kwargs):
...
+                            " rolling back configuration."
+                        )
                         ret["out"] = True
                     except Exception as exception:  # pylint: disable=broad-except
-                        ret[
-                            "message"
-                        ] = 'Commit check passed, but skipping commit for dry-run andi while rolling back configuration exception occurred "{}"'.format(
-                            exception
+                        ret["message"] = (
+                            "Commit check passed, but skipping commit for dry-run andi"
```
Comment (garethgreenaway) [inline salt/modules/junos.py:1237]: Not a blocker but a minor typo we could fix while making changes here.

Comment (s0undt3ch) [reply salt/modules/junos.py]: Fixed

File: salt/config/schemas/esxvm.py (hunk 33 above)
Comment (garethgreenaway) [reply salt/config/schemas/esxvm.py:309]: Yup!  Missed the space at the end on the first line.

File: salt/utils/parsers.py [hunk 36]
```diff
@@ -3210,15 +3241,19 @@ def _mixin_setup(self):
             "--min-extra-modules",
             dest="min_extra_mods",
             default=None,
-            help="One or comma-separated list of extra Python modules"
-            "to be included into Minimal Salt.",
+            help=(
+                "One or comma-separated list of extra Python modules"
```
Comment (garethgreenaway) [inline salt/utils/parsers.py:3245]: Do we need a space at the end here?

File: salt/utils/parsers.py [hunk 37]
```diff
@@ -3210,15 +3241,19 @@ def _mixin_setup(self):
...
+            ),
         )
         self.add_option(
             "--thin-extra-modules",
             dest="thin_extra_mods",
             default=None,
-            help="One or comma-separated list of extra Python modules"
-            "to be included into Thin Salt.",
+            help=(
+                "One or comma-separated list of extra Python modules"
```
Comment (garethgreenaway) [inline salt/utils/parsers.py:3254]: Do we need a space at the end here?

File: salt/utils/parsers.py [hunk 38]
```diff
@@ -3380,8 +3427,10 @@ def _mixin_setup(self):
             dest="ssh_update_roster",
             default=False,
             action="store_true",
-            help="If hostname is not found in the roster, store the information"
-            "into the default roster file (flat).",
+            help=(
+                "If hostname is not found in the roster, store the information"
```
Comment (garethgreenaway) [inline salt/utils/parsers.py:3431]: Do we need a space at the end here?

File: salt/utils/parsers.py (hunk 36 above)
Comment (s0undt3ch) [reply salt/utils/parsers.py:3245]: Fixed

File: salt/utils/parsers.py (hunk 38 above)
Comment (s0undt3ch) [reply salt/utils/parsers.py:3431]: Fixed

File: salt/utils/parsers.py (hunk 37 above)
Comment (s0undt3ch) [reply salt/utils/parsers.py:3254]: Fixed

File: tests/integration/modules/test_mac_brew_pkg.py [hunk 39]
```diff
@@ -74,31 +74,24 @@ def test_version(self):
...
                 )
                 self.assertIn(
                     ADD_PKG,
                     pkg_list,
                     msg=(
-                        "package: {} is not in\
-                              the list of installed packages: {}".format(
-                            ADD_PKG, pkg_list
-                        )
+                        "package: {} is not in                              the list of"
```
Comment (garethgreenaway) [inline tests/integration/modules/test_mac_brew_pkg.py:85]: Extra white space here.

Comment (s0undt3ch) [reply tests/integration/modules/test_mac_brew_pkg.py:85]: Fixed

**Comment by s0undt3ch** [comment]
re-run full all

**Review by Ch3LL — DISMISSED** [review]

File: tests/support/napalm.py [hunk 40]
```diff
@@ -364,12 +364,17 @@
             "level": 15,
             "password": "$1$0P70xKPa$4jt5/10cBTckk6I/w/",
             "sshkeys": [
-                "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC4pFn+shPwTb2yELO4L7NtQrKOJXNeCl1je\
-            l9STXVaGnRAnuc2PXl35vnWmcUq6YbUEcgUTRzzXfmelJKuVJTJIlMXii7h2xkbQp0YZIEs4P\
-            8ipwnRBAxFfk/ZcDsdfsdfsdfsdN56ejk345jhk345jk345jk341p3A/9LIL7l6YewLBCwJj6\
-            D+fWSJ0/YW+7oH17Fk2HH+tw0L5PcWLHkwA4t60iXn16qDbIk/ze6jv2hDGdCdz7oYQeCE55C\
-            CHOHMJWYfN3jcL4s0qv8/u6Ka1FVkV7iMmro7ChThoV/5snI4Ljf2wKqgHH7TfNaCfpU0WvHA\
-            nTs8zhOrGScSrtb mircea@master-roshi"
+                "ssh-rsa"
```
Comment (garethgreenaway) [inline tests/support/napalm.py:367]: Not sure if this is correct.  I don't think we want those white space lines in there, the SSH key should be one long continuous line right?

Comment (s0undt3ch) [reply tests/support/napalm.py:367]: Nice find. Will be fixed on next push.

File: tests/unit/modules/test_drbd.py [hunk 41]
```diff
@@ -57,8 +59,10 @@ def test_overview(self):
             "synchronisation: ": "syncbar",
         }
         mock = MagicMock(
-            return_value="Salt:Stack True master/minion \
-        UpToDate/partner syncbar None 50 50"
+            return_value=(
+                "Salt:Stack True master/minion         UpToDate/partner syncbar None"
```
Comment (garethgreenaway) [inline tests/unit/modules/test_drbd.py:63]: Extra white space here?

File: tests/unit/modules/test_drbd.py [hunk 42]
```diff
@@ -39,8 +39,10 @@ def test_overview(self):
             "used": "50",
         }
         mock = MagicMock(
-            return_value="Salt:Stack True master/minion \
-        UpToDate/UpToDate True None 50 50 666 888"
+            return_value=(
+                "Salt:Stack True master/minion         UpToDate/UpToDate True None 50"
```
Comment (garethgreenaway) [inline tests/unit/modules/test_drbd.py:43]: Extra white space here?

Comment (s0undt3ch) [reply tests/unit/modules/test_drbd.py:43]: Will be fixed on next push

File: tests/unit/modules/test_drbd.py (hunk 41 above)
Comment (s0undt3ch) [reply tests/unit/modules/test_drbd.py:63]: Will be fixed on next push

File: tests/unit/test_module_names.py [hunk 43]
```diff
@@ -91,8 +91,14 @@ def test_module_name(self):
                 path, directory, filename.split("_test")[0]
             )
 
-        error_msg += "\nIf you believe one of the entries above should be ignored, please add it to either\n"
-        error_msg += "'EXCLUDED_DIRS' or 'EXCLUDED_FILES' in 'tests/unit/test_module_names.py'.\n"
+        error_msg += (
```
Comment (garethgreenaway) [inline tests/unit/test_module_names.py:94]: Not a blocker for this but could we just combine all these lines into one addition?

Comment (s0undt3ch) [reply tests/unit/test_module_names.py:94]: Will be fixed on next push

**Comment by s0undt3ch** [comment]
All fixes pushed

**Review by garethgreenaway — DISMISSED** [review]

**Comment by s0undt3ch** [comment]
re-run full all

File: salt/modules/mine.py [hunk 44]
```diff
@@ -44,8 +44,7 @@ def _auth():
             __context__["auth"] = salt.crypt.SAuth(__opts__)
         except SaltClientError:
             log.error(
-                "Could not authenticate with master."
-                "Mine data will not be transmitted."
+                "Could not authenticate with master.Mine data will not be transmitted."
```
Comment (twangboy) [inline salt/modules/mine.py:47]: `master. Mine`

File: salt/modules/win_lgpo.py [hunk 45]
```diff
@@ -10134,8 +10501,8 @@ class in this module.
                                     )
                                     if not _ret:
                                         msg = (
-                                            "An error occurred attempting to remove previously"
-                                            "configured users with right {}."
+                                            "An error occurred attempting to remove"
+                                            " previouslyconfigured users with right {}."
```
Comment (twangboy) [inline salt/modules/win_lgpo.py:10505]: `previously configured`

**Review by twangboy — CHANGES_REQUESTED** [review]
Got to salt/states/acme.py

File: salt/modules/mine.py (hunk 44 above)
Comment (s0undt3ch) [reply salt/modules/mine.py:47]: Will be fixed on next push

File: salt/modules/win_lgpo.py (hunk 45 above)
Comment (s0undt3ch) [reply salt/modules/win_lgpo.py:10505]: Will be fixed on next push

**Comment by s0undt3ch** [comment]
re-run full all

**Review by Ch3LL — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

File: tests/salt-tcpdump.py [hunk 46]
```diff
@@ -66,7 +66,7 @@ def add_args(self):
             default="eth0",
             dest="iface",
             required=False,
-            help=("the interface to dump the" "master runs on(default:eth0)"),
+            help="the interface to dump themaster runs on(default:eth0)",
```
Comment (twangboy) [inline tests/salt-tcpdump.py:69]: `the master`

File: tests/support/zfs.py [hunk 47]
```diff
@@ -197,9 +213,11 @@ def __init__(self):
                     "        groupquota@...  YES       NO   <size> | none",
                     "        written@<snap>   NO       NO   <size>",
                     "",
```
Comment (twangboy) [inline tests/support/zfs.py:215]: We should probably not blacken this entire section... it makes it far less readable

**Comment by s0undt3ch** [comment]
🎉

**Review by twangboy — APPROVED** [review]

**Pull request merged**